Libraries/JavaScriptAppEngine/polyfills/document.js

```diff
@@ -1,34 +1,32 @@
-/* eslint global-strict: 0 */
-(function(GLOBAL) {
-  /**
-   * The document must be shimmed before anything else that might define the
-   * `ExecutionEnvironment` module (which checks for `document.createElement`).
-   */
+/* eslint strict: 0 */
+/**
+ * The document must be shimmed before anything else that might define the
+ * `ExecutionEnvironment` module (which checks for `document.createElement`).
+ */
```

Comment:
:+1: 

will clean this up in a follow up – @janicduplessis, can you just remove the comment for now, please? It’s misleading